hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -141,6 +148,10 @@ public boolean useFileListingMetadata() {
     return getBoolean(METADATA_ENABLE_PROP);
```

Comment:
ideally we should rename this method

Reply:
Yes, the name does not fit other such methods in HUDI.